build-phar.sh

```diff
@@ -1,18 +1,17 @@
-#!/usr/bin/env bash
-
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash --pure
+#! nix-shell -p bash phpPackages.box phpPackages.composer
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/fae7198d73783051aae436f84bb91bd9932a6f6d.tar.gz
```

Comment:
Eh, now we moved a random hash in here? 😬

Reply:
Let me delete that script, it's not needed any more

Reply:
It was needed in the first method, but since I switched to flake, it's not needed any more.